pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -1974,6 +1975,26 @@ The delayed message index bucket time step(in seconds) in per bucket snapshot se
             doc = "The threshold to triggering automatic offload to long term storage"
     )
     private long managedLedgerOffloadThresholdInSeconds = -1L;
+    @FieldContext(
+            category = CATEGORY_STORAGE_OFFLOADING,
+            doc = "The number of bytes of namespaces before triggering automatic offload to long term storage"
+    )
+    private Map<String, Long> managedLedgerOffloadNamespaceThresholdInBytes;
+    @FieldContext(
+            category = CATEGORY_STORAGE_OFFLOADING,
+            doc = "The number of bytes of topics before triggering automatic offload to long term storage"
+    )
+    private Map<String, Long> managedLedgerOffloadTopicThresholdInBytes;
+    @FieldContext(
+            category = CATEGORY_STORAGE_OFFLOADING,
+            doc = "The number of seconds of namespaces before triggering automatic offload to long term storage"
+    )
+    private Map<String, Long> managedLedgerOffloadNamespaceThresholdInSeconds;
+    @FieldContext(
+            category = CATEGORY_STORAGE_OFFLOADING,
+            doc = "The number of seconds of topics before triggering automatic offload to long term storage"
+    )
+    private Map<String, Long> managedLedgerOffloadTopicThresholdInSeconds;
```

Comment:
I don't think that it is a good idea to write staticaly topics or namespaces in broker.conf.

Usually namespace and topic configuration are dynamic 

Why are you taking this approach?

Reply:
@eolivelli How about make it to dynamic config?